modules/test/playwright/tests/frontend-data-set-views-web/pages/ActionsPage.ts

```diff
@@ -49,9 +49,13 @@ export class ActionsPage {
 
 	async goto() {
 		await this.viewsPage.goto();
-		await this.viewsPage.gotoSampleDataSetView().then(() => {
-			this.page.getByRole('button', {name: 'Actions'}).first().click();
-		});
+		await this.viewsPage.gotoSampleDataSetView();
```

Comment:
[not in scope of PR, but...] In `client-extensions-web` I preferred structure that ends up in singe `goto*`, like [here](https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/tests/client-extension-web/pages/ClientExtensionsPage.ts#L39-L48). In this case, it would be just `viewsPage.gotoSampleDataSetView()`. Is there a benefit to separate `goto`?